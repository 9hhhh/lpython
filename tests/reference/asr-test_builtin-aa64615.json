{
    "basename": "asr-test_builtin-aa64615",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/test_builtin.py",
    "infile_hash": "edd7f1dba3727bb7bfca6f71c13c89ac86e3615a98a749c47067ed53",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-test_builtin-aa64615.stdout",
    "stdout_hash": "ab7db23fa97b9892d20fa26134e65870f8f9fe65a5e5392e342fd0f1",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}