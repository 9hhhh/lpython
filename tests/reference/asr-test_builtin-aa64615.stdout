(TranslationUnit (SymbolTable 1 {_lfortran_main_program: (Subroutine (SymbolTable 9 {}) _lfortran_main_program [] [(SubroutineCall 1 test_ord () [] ())] Source Public Implementation () .false. .false.), lpython_builtin: (IntrinsicModule lpython_builtin), main_program: (Program (SymbolTable 8 {}) main_program [] [(SubroutineCall 1 _lfortran_main_program () [] ())]), test_ord: (Subroutine (SymbolTable 2 {i: (Variable 2 i Local () () Default (Integer 4 []) Source Public Required .false.), ord: (ExternalSymbol 2 ord 4 ord lpython_builtin [] ord Public), s: (Variable 2 s Local () () Default (Character 1 -2 () []) Source Public Required .false.)}) test_ord [] [(= (Var 2 s) (ConstantString "1" (Character 1 1 () [])) ()) (= (Var 2 i) (FunctionCall 2 ord () [(Var 2 s)] [] (Integer 4 []) () ()) ()) (Print () [(Var 2 i)])] Source Public Implementation () .false. .false.)}) [])
